samples/gibbs_ensemble/run_for_several_kT.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```
For portability reasons, we prefer Dash over Bash whenever possible. In this case, the script can be ported to Dash by simply changing the conditional at line 6.